lib/components/Checkbox.js

```diff
@@ -1,44 +1,41 @@
 import React from "react";
 import classnames from "classnames";
 import { v4 as uuid } from "uuid";
+import Label from "./Label";
 
 const Checkbox = ({
   id = "",
-  className,
+  className = "",
   name,
-  label,
-  required,
-  dataTestId,
+  label = "",
+  required = false,
   ...otherProps
 }) => {
   const uniqueId = uuid();
 
   return (
-    <>
-      <div className="relative flex items-start">
-        <div className="flex items-center h-5">
-          <input
-            name={name}
-            type="checkbox"
-            className={classnames([
-              "w-4 h-4 text-purple-500 transition duration-200 ease-in-out form-checkbox cursor-pointer",
-              className
-            ])}
-            data-test-id={dataTestId}
-            id={id || uniqueId}
-            {...otherProps}
-          />
-        </div>
-        {(label || required) && (
-          <label
-            htmlFor={id || uniqueId}
-            className="mb-0 ml-2 text-sm font-medium leading-5 text-gray-700"
-          >
-            {label || (required ? <span className="text-red-700">*</span> : "")}
-          </label>
-        )}
+    <div className="relative flex flex-row items-center justify-start">
+      <div className="flex flex-row items-center justify-center h-5">
+        <input
+          name={name}
+          type="checkbox"
+          className={classnames([
+            "w-4 h-4 text-purple-500 transition duration-200 ease-in-out form-checkbox cursor-pointer",
+            className
+          ])}
+          id={id || uniqueId}
+          {...otherProps}
+        />
       </div>
-    </>
+      {(label || required) && (
+        <Label
+          htmlFor={id || uniqueId}
+          className="mb-0 ml-2 text-sm font-medium leading-5 text-gray-700"
+        >
+          {label || (required ? <span className="text-red-700">*</span> : "")}
```

Comment:
@yedhink _a
Please remove red color for required. We are changing it into black in all the places.
```suggestion
          {label || (required ? <span className="ml-0.5">*</span> : "")}
```

Reply:
required is handled inside the Label component right. So we just need to pass the prop.